component/mcsd/component.go

```diff
@@ -26,6 +26,9 @@ var _ component.Lifecycle = &Component{}
 var rootDirectoryResourceTypes = []string{"Organization", "Endpoint"}
 var defaultDirectoryResourceTypes = []string{"Organization", "Endpoint", "Location", "HealthcareService", "PractitionerRole", "Practitioner"}
 
+// ParentOrganizationMap maps parent organizations (with URA identifier) to their linked child organizations
+type ParentOrganizationMap map[*fhir.Organization][]*fhir.Organization
```

Comment:
You're only using this type within this package, so no need to export the type (so don't ca[pitaize the first letter of Parent)